src/ic_app.cpp

```diff
@@ -23,15 +22,14 @@ namespace
     Config _appConfig;
     bool _appIsRunning = false;
     bool _appIsExiting = false;
+    RendererConfig _rendererConfig{};
```

Comment:
Feels weird to be holding on to the config, the config data we pass in to the Rendered for initialization should then by copied into the Renderer

Reply:
Ultimately this wasn't going to last, but adjusted.